<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lenzeToolUI</class>
 <widget class="QWidget" name="lenzeToolUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,0,0,0);
background-image:url(://Service_Tools/Service_Tools/lenze_background.png);
border: 0px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>105</y>
      <width>66</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://Service_Tools/Service_Tools/lenze_driver.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>10</y>
      <width>616</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">Service Panel/Tools/Lenze motor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="di3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="di2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="di4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="di5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="an1">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">---(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="an2">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>245</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">---(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>425</y>
      <width>741</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,30);
background-image:url(://transparent.png);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="vpower_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>3</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);
background-color:rgb(0,0,0,0);</string>
     </property>
     <property name="text">
      <string notr="true">ERRORS:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="errDiagnostic">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>2</y>
       <width>626</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);
background-color:rgb(0,0,0,0);</string>
     </property>
     <property name="text">
      <string notr="true">ssss</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="errInternal">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>19</y>
       <width>626</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);
background-color:rgb(0,0,0,0);</string>
     </property>
     <property name="text">
      <string notr="true">ssss</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="di1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="do1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="sto">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>305</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(://yes.png);
background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="vbus">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>158</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vbus_2">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>175</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">THRESH HIGH (%):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vbus_3">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>195</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">THRESH LOW (%):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vbus_4">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>155</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">VBUS (VAC):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="thh">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>176</y>
      <width>36</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="thl">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>196</y>
      <width>36</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="templab">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>137</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">TEMP (°C):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temp">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
